runtime/tests/checker/attachments_test.go

```diff
@@ -540,35 +540,12 @@ func TestCheckAttachmentWithMembers(t *testing.T) {
 				init(x: @R) {
 					self.x <- x
 				}
-				destroy() {
-					destroy self.x
-				}
 			}`,
 		)
 
 		require.NoError(t, err)
 	})
 
-	t.Run("resource field no destroy", func(t *testing.T) {
```

Comment:
I guess this is removed because the test above now is equivalent and tests the same?